src/js/scope_events.js

```diff
@@ -195,7 +195,7 @@
 		fireEvent('change', handleNumber);
 
 		if ( options.events.snap ) {
-			start(event, { handles: [scope_Handles[total]] });
+			start(event, { handles: [scope_Handles[handleNumber]] });
```

Comment:
Right, got it now. [Merge progress](https://github.com/leongersen/noUiSlider/tree/merge).